_posts/2020-18-06-Triplestores-Onotologies-Reasoning.md

```diff
@@ -340,15 +351,30 @@ take a very long time to complete.
 
 The foregoing discussion has introduced the reader to Triplestores, a
 kind of graph database which is used to store and query RDF.  This led
-to a discussion of OWL 2 logical inferrencing and topics that a user
+to a discussion of OWL 2 logical inferencing and topics that a user
 who wants to write queries will need to know before attempting to use
 querying in Triplestores.  OWL 2 in conjunction with RDF gives a
 powerful set of tools with which to model the medieval Irish
-genealogical tradition.  The ability of OWL 2 to logically reason
-about the data to create new data substantially expands and transforms
-that data in ways that a team of humans who are experts in the
-medieval Irish genealogies would not be able to do affordably.
-
-While the actual act of querying will be deferred until a future
-discussion, the reader will now be equipped to understand the whole
-context for querying the IrishGen before handling SPARQL specifically.
+genealogical tradition of which IrishGen is a concrete example. A note
+here that a user who is excited by the possibilities afforded by OWL 2
+will need data to model before making the attempt.  For instance, an
+OWL 2 data model and RDF encoding for the medieval Irish annals in
+their various MS contexts would be a very good place to start if one
+is looking for a project of this type.
+
+While powerful, there are downsides to inferencing of this type.
+First, the reasoner will make inferences without respect to the
+plausability of the data or the outcome.  Second, it will make
+inferences withou consideration for the weight of evidence.  If the
+system detects a logical connection, it will assume that it is true as
+it assumes all assertions contained in the data are true.  Third,
+importantly for backwards-chaining systems, query response times can
+be frustrating.  Caution and a critical view of any data presented is
+always a good policy and no less so here.  While these are drawbacks
+as there are drawbacks to any system, it does not diminish the
+usefulness of the system for many users.
```

Comment:
Without wanting to get into an endless loop of advantages and disadvantages, one could point out that treating each assertion is true is actually useful for critical comparison of different historical sources.